fastlane/Fastfile

```diff
@@ -283,6 +286,56 @@ platform :ios do
     trigger_release_build(branch_to_build: "release/#{version}")
   end
 
+  #####################################################################################
+  # update_metadata_on_app_store_connect
+  # -----------------------------------------------------------------------------------
+  # Uploads the localized metadata tracked in the project repository to App Store
+  # Connect, optionally updating screenshots, too.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane update_metadata_on_app_store_connect [with_screenshots:<with screenshots>]
+  #
+  # Example:
+  # bundle exec fastlane update_metadata_on_app_store_connect
+  # bundle exec fastlane update_metadata_on_app_store_connect with_screenshots:true
+  #####################################################################################
+  desc 'Updates the App Store Connect localized metadata'
+  lane :update_metadata_on_app_store_connect do |options|
+    update_wordpress_metadata_on_app_store_connect(options)
+    update_jetpack_metadata_on_app_store_connect(options)
+  end
+
+  #####################################################################################
+  # update_wordpress_metadata_on_app_store_connect
+  # -----------------------------------------------------------------------------------
+  # This lane reads the information from `fastlane/metadata/` and uploads it to
+  # App Store Connect for the version specified in the version xcconfig file.
+  #
+  # Optionally, it can also upload new screenshots, overriding the existing ones.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane update_wordpress_metadata_on_app_store_connect
+  #
+  # Example:
+  # bundle exec fastlane update_wordpress_metadata_on_app_store_connect
+  # bundle exec fastlane update_wordpress_metadata_on_app_store_connect with_screenshots:true
+  #####################################################################################
+  desc 'Uploads the WordPress metadata to App Store Connect, localized, and optionally including screenshots.'
+  lane :update_wordpress_metadata_on_app_store_connect do |options|
+    # Skip screenshots by default. The naming is "with" to make it clear that
+    # callers need to opt-in to adding screenshots. The naming of the deliver
+    # parameter, on the other hand, uses the skip verb.
+    with_screenshots = options.fetch(:with_screenshots, false)
+    skip_screenshots = with_screenshots == false
+
+    upload_to_app_store(
+      **COMMON_UPLOAD_TO_APPSTORE_PARAMS,
```

Comment:
nit💄:  I know that I'm the one who suggested this constant name 😅  but now that I read it I'm thinking maybe it should be spelled `UPLOAD_TO_APPSTORE_COMMON_PARAMS`… 🤷